packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -275,73 +355,69 @@ body {
       display: none;
     }
   }
-  .aa-SourceNoResults {
-    padding: var(--aa-spacing);
-  }
   &:empty {
     // hide empty section
     display: none;
   }
-  // list of results inside the source
-  .aa-List {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    position: relative;
+  // See all button
+  .aa-SourceFooterSeeAll {
+    background: linear-gradient(
+      180deg,
+      var(--aa-background-color),
+      rgba(128, 126, 163, 0.14)
+    );
+    border: 1px solid var(--aa-muted-color);
+    border-radius: 5px;
+    box-shadow: inset 0 0 2px #fff, 0 2px 2px -1px rgba(76, 69, 88, 0.15);
+    color: inherit;
+    font-size: 0.95em;
+    font-weight: 500;
```

Comment:
If we go with the inherited font as @haroen suggested and the inherited font doesn't support a `500` weight (e.g., they have `400` and `600`), it won't be properly styled. We should probably make this a variable.

Reply:
What's the status on this one?